playground/src/markdown/Explainer.md

```diff
@@ -0,0 +1,81 @@
+# What the f*ck? A short explainer
```

Comment:
It seems none of the headers have anchors, that makes sharing documentation harder. It would be nice to have a table of contents and anchors like we have in the mdbook

Reply:
I think this is not supported by `react-markdown`. Normally `#` should create an anchor and and ID. 

Added some space for all the pages